scripts/validate_docstrings.py

```diff
@@ -690,8 +691,14 @@ def get_validation_data(doc):
                 errs.append(error('PR09', param_name=param))
 
     if doc.is_function_or_method:
+        (tree, ) = ast.parse(doc.method_source.strip()).body
```

Comment:
Is this required to have the left hand side of the assignment look like this?

Reply:
the ast.parse returns a list, so it's a way of taking the first element.

I have replaced it for ...body[0], which I think is more common